src/k8s-engine/k8s.ts

```diff
@@ -29,6 +29,15 @@ export class KubernetesError extends Error {
   readonly fatal: boolean;
 }
 
+export class KimBuilderInstallError extends Error {
+  readonly fatal: boolean;
+  constructor(titlePart: string, message: string) {
+    super(message);
+    this.name = titlePart;
+    this.fatal = false;
```

Comment:
Nothing actually ever sets fatal to be true (because there's only one place we construct an instance of `KimBuilderInstallError`).